### PR TITLE
Change port 5000 to 5001 to avoid clash with macos

### PR DESCRIPTION
**Pull request by benfoley**
MacOS 12 has a new Airplay speaker sharing feature which listens to port 5000. It is on by default, and causes an error when running Docker on port 5000. 

```
➜ docker run --rm -p 5000:5000/tcp coedl/elpis:latest

docker: Error response from daemon: Ports are not available: listen tcp 0.0.0.0:5000: bind: address already in use.
ERRO[0000] error waiting for container: context canceled
```

This PR changes the port used for the container to 5001, while keeping the Flask dev server on 5000. Main changes are in the docs where `-p 5000:5000` changes to `-p 5001:5000` with accompanying instruction to browse to `0.0.0.0:5001`

Would it be better to change the Flask server to run on 5001 as well, or is it OK to leave that on 5000? 

Also, removed the docs about developing in VS Code because they are well outdated.

### CODE REVIEW

File: gpu/Dockerfile [hunk 1]
```diff
@@ -309,5 +309,5 @@ WORKDIR /elpis
 
 ENTRYPOINT ["poetry", "run", "flask", "run", "--host", "0.0.0.0"]
 
-EXPOSE 5000:5000
-EXPOSE 3000:3000
+EXPOSE 5000
```
Comment (mattchrlw) [inline gpu/Dockerfile:312]: i think this should be 5001

**Review by mattchrlw — CHANGES_REQUESTED** [review]
I think it would be good to change the Flask port as well.

File: gpu/Dockerfile (hunk 1 above)
Comment (benfoley) [reply gpu/Dockerfile]: done

**Comment by benfoley** [comment]
Flask port changed now too

**Review by mattchrlw — APPROVED** [review]
lgtm!

**Pull request merged**